frontend/src/component/ContentListItem.jsx

```diff
@@ -21,6 +21,7 @@ require('./ContentListItem.styl')
 const numberCommentsTitle = (numberComments, t) => {
```

Comment:
It seems that i18next can manage plurals, even with arabic: https://www.i18next.com/translation-function/plurals
Shouldn't we use this instead of re-designing a solution?